concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-278	278	jalar	pull	PULL	1455	Verb	4.09803921568627	5	2	
+Desrochers-2010-330-279	279	jamar	polish off			Verb	2.31372549019608	5		
+Desrochers-2010-330-280	280	lamer	lick	LICK	319	Verb	4.66666666666667	5	6	
+Desrochers-2010-330-281	281	laminar	laminar			Verb	2.72549019607843	7	4	
+Desrochers-2010-330-282	282	lanzar	launch			Verb	5.50980392156863	6	93	5.28
+Desrochers-2010-330-283	283	lavar	wash	WASH	1453	Verb	6.41176470588235	5	35	6.35
+Desrochers-2010-330-284	284	limpiar	clean	CLEAN (SOMETHING)	2235	Verb	6.63725490196079	7	57	6.49
+Desrochers-2010-330-285	285	lustrar	polish	POLISH (SOMETHING)	3659	Verb	2.42156862745098	7	6	
+Desrochers-2010-330-286	286	marcar	dial			Verb	5.87254901960784	6	69	6.03
+Desrochers-2010-330-287	287	matar	kill	KILL	1417	Verb	5.92156862745098	5	239	5.52
+Desrochers-2010-330-288	288	meter	put	PUT	998	Verb	6.4159747889007	5	130	6.68
+Desrochers-2010-330-289	289	mezclar	mix	MIX	2017	Verb	5.82352941176471	7	24	6.12
+Desrochers-2010-330-290	290	mitigar	mitigate			Verb	2.95098039215686	7	6	
+Desrochers-2010-330-291	291	mondar	peel	PEEL	275	Verb	3.62745098039216	6	1	
+Desrochers-2010-330-292	292	mostrar	show	SHOW	1039	Verb	5.93	7	119	6.07
+Desrochers-2010-330-293	293	mutilar	mutilate			Verb	3.86274509803922	7	4	
+Desrochers-2010-330-294	294	nivelar	level			Verb	3.96078431372549	7	3	
+Desrochers-2010-330-295	295	nutrir	feed	FEED	739	Verb	3.84313725490196	6	5	
+Desrochers-2010-330-296	296	obsequiar	give away			Verb	3.93137254901961	9	2	
+Desrochers-2010-330-297	297	ofuscar	obfuscate			Verb	3.02941176470588	7		
+Desrochers-2010-330-298	298	ornar	decorate	DECORATE	1739	Verb	2	5	7	
+Desrochers-2010-330-299	299	pagar	pay	PAY	718	Verb	6.84	5	342	6.75
+Desrochers-2010-330-300	300	parar	stop	STOP DOING	3351	Verb	6.47058823529412	5	200	6.64
+Desrochers-2010-330-301	301	partir	share			Verb	5.88235294117647	6	79	5.78
+Desrochers-2010-330-302	302	pasar	go to			Verb	6.37254901960784	5	1040	6.33
+Desrochers-2010-330-303	303	pelar	peel	PEEL	275	Verb	6.04901960784314	5	12	
+Desrochers-2010-330-304	304	picar	click			Verb	5.92156862745098	5	14	
+Desrochers-2010-330-305	305	pintar	paint	PAINT (SOMETHING)	3122	Verb	6.23529411764706	6	47	6.4
+Desrochers-2010-330-306	306	pisar	step on	TREAD	575	Verb	6.17647058823529	5	41	6.63
+Desrochers-2010-330-307	307	portar	retrieve			Verb	3.30392156862745	6	7	
+Desrochers-2010-330-308	308	posar	pose			Verb	4.77450980392157	5	13	
+Desrochers-2010-330-309	309	pulir	polish	POLISH (SOMETHING)	3659	Verb	3.90196078431373	5	5	
+Desrochers-2010-330-310	310	quebrantar	violate			Verb	3.52941176470588	10	7	
+Desrochers-2010-330-311	311	quitar	remove			Verb	6.26470588235294	6	76	5.28
+Desrochers-2010-330-312	312	rajar	crack	SPLIT OPEN	3080	Verb	4.80392156862745	5	1	
+Desrochers-2010-330-313	313	rascar	scratch	SCRATCH	1436	Verb	5.53921568627451	6	9	
+Desrochers-2010-330-314	314	romper	break	BREAK (DESTROY OR GET DESTROYED)	680	Verb	6.26470588235294	6	199	6.39
```

Comment:
romper is transitive

Reply:
Indeed, but not sure if 2558 | BREAK (CLEAVE) would be better given the definition to BREAK (DESTROY OR GET DESTROYED)

Reply:
What's the issue with the definition?
I'm not sure about the exact meaning of _romper_, but would TEAR (SHRED) fit the meaning better?

Reply:
TEAR (SHRED) could be "romper", but I suppose something like "destrozar" or "partir" is closer to "to tear".

As for the definition, I meant that BREAK (DESTROY) is defined as "to divide or be divided", i.e., it would fit a transitive verb.

Reply:
leave unliked otherwise, but maybe specify the meaning in English: break (something)